client/tests/functional/details/CurlConnectionTests.cpp

```diff
@@ -255,7 +255,7 @@ TEST("Testing CurlConnection()")
         REQUIRE_THROWS_CODE(connection->Post(url + "/names", "{}"), HttpNotFound);
 
         url = server.GetBaseUrl() + "/api/v1/contents/" + c_instanceId + "/namespaces/" + c_namespace + "/names/" +
-              c_productName + "/versYions/" + c_version + "/files?action=GenerateDownloadInfo";
+              c_productName + "/versions_/" + c_version + "/files?action=GenerateDownloadInfo";

```

Comment:
I'm assuming this is functionally equivalent, but it has the advantage of not tripping up spell checking tools.

Reply:
it does, but this was meant to be obvious that something is wrong. The _ is a bit more discrete

Reply:
Fair. How about moving the `Y` to the end so you have `version` (which is a word) + `Ys` which is short enough that at least check-spelling won't flag it (two letter tokens just aren't worth checking).

Reply:
Sounds good